api/src/pcapi/core/educational/adage_backends/adage.py

```diff
@@ -29,11 +33,19 @@ def __init__(self) -> None:
 
     def notify_prebooking(self, data: prebooking.EducationalBookingResponse) -> None:
         api_url = f"{self.base_url}/v1/prereservation"
-        api_response = requests.post(
-            api_url,
-            headers={self.header_key: self.api_key, "Content-Type": "application/json"},
-            data=data.json(),
-        )
+        try:
+            api_response = requests.post(
+                api_url,
+                headers={self.header_key: self.api_key, "Content-Type": "application/json"},
+                data=data.json(),
+            )
+        except ConnectionError as exp:
+            logger.info(traceback.format_exc)
```

Comment:
un peu de contexte peut être ?

logger.info("could not connect to adage, error: %s", traceback.format_exc())